src/browserlib/map-ids-to-headings.mjs

```diff
@@ -36,6 +36,18 @@ export default function () {
   const { outline, nodeToSection } = createOutline(document.body);
   const sections = flattenSections(outline);
 
+  // Compute once whether we created a single page version out of multiple pages
+  const isMultipage = !!document.querySelector('[data-reffy-page]');
+
+  function getAbsoluteUrl(node) {
+    const page = isMultipage ?
```

Comment:
should this be factored out since that function is repeated 4 times?

Reply:
Done.